zerver/tornado/handlers.py

```diff
@@ -164,7 +164,7 @@ def get(self, *args: Any, **kwargs: Any) -> None:
             self.set_header(h[0], h[1])
 
         if not hasattr(self, "_new_cookies"):
-            self._new_cookies = []  # type: List[http.cookie.SimpleCookie]
+            self._new_cookies = []  # type: List[http.cookie.SimpleCookie[str]]
         self._new_cookies.append(response.cookies)
```

Comment:
Incidentally, does this block do anything? Tornado hasn’t had a “`_new_cookies`” since tornadoweb/tornado@70ea7a5a6ca956b613168c9d123ee7cc951054bf in Tornado 2.3.0 (released 2012-05-31).

Reply:
Interesting; worth investigating!  The structurally intend of that class is that the `get_response` is based on the code of that class in Django upstream, and everything else is the minimal Tornado shim to have things run.  We've been pretty careful about updating `get_response` when we upgrade Django itself, but I'm not aware of our doing similar work for Tornado upgrades.  So it's likely that code is dead; it came in the very first Tornado commit during the very first month of Zulip development: 9afd63692f95c502edf67ee03beb186eca81559a.  

It's unclear to me whether there's any other changes we should make there, but if I had to guess, we can just delete that code because our Tornado code isn't meant to have anything but read access to cookies anyway?  Probably worth forking as a separate discussion issue.